src/sentry/sentry_metrics/consumers/indexer/batch.py

```diff
@@ -71,16 +72,24 @@ def invalid_metric_tags(tags: Mapping[str, str]) -> Sequence[str]:
     return invalid_strs
 
 
+def extract_use_case_id(mri: str) -> Optional[str]:
+    use_case_mapping = {"transactions": "performance", "sessions": "release-health"}
+    if (matched := MRI_RE_PATTERN.match(mri)) is not None:
+        use_case_str = matched.group(2)
```

Comment:
this is a minor thing, but I think it could help if we could rename `use_case_str` here to something like `namespace` since that's what we're getting from the MRI